backend/src/main/java/quarano/Quarano.java

```diff
@@ -71,6 +74,14 @@ PasswordEncoder getPasswordEncoder() {
 		return new BCryptPasswordEncoder();
 	}
 
+	@Bean
+	FlywayConfigurationCustomizer getFlywayCustomizer(DepartmentProperties prop) {
+		return configuration -> configuration
+				.locations(
+						configuration.getLocations()[0],
+						new Location("classpath:db/client_migration/" + prop.getDefaultDepartment().getRkiCode() + "RKI"));
```

Comment:
We could just remove the trailing dot from the RKI code itself, couldn't we?

Reply:
I didn't want to change the path again after we set up Cuxhaven. It remained unclear if we had problems with Flyway in the Mannheim stage because we changed the path.

Reply:
As long as it find's the same migration files it should still recognize the database is already on a particular version. But no problem, we can just go as suggested.